modules/claims_api/app/swagger/claims_api/forms/form_0966_v0_example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "form/0966",
+  "attributes": {
+    "type": "compensation",
+    "participant_claimant_id": 123456789,
+    "participant_vet_id": 987654321,
```

Comment:
i don't think this matters, but, above, I see that this is usually the same number

Reply:
it should be fine for example purposes.

Reply:
we want this to be an integer after all.